Assignments/src/com/java/assignment6/VampireNumber.java

```diff
@@ -0,0 +1,80 @@
+package com.java.assignment6;
+
+import java.util.Scanner;
+
+public class VampireNumber {
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        int v=0;
+        long number,i,c=0,f=0,j=0,x,k,p,c1,c2,b[]=new long[100];
+        System.out.println("Enter a number:");
```

Comment:
can we add meaningful names instead of x,k,p....

Reply:
yeah sure